src/Data/Functor/Linear/Internal/Applicative.hs

```diff
@@ -9,9 +13,10 @@ where
 
 import qualified Control.Monad.Trans.Reader as NonLinear
 import Data.Functor.Compose
-import Data.Functor.Const
+import Data.Functor.Const (Const (..))
```

Comment:
When there is a conflict, I'd rather us use `hiding (conflicting thing)`. It's more robust.

Reply:
There seems to be no conflict after all